src/lib/helpers/jsToAqua.ts

```diff
@@ -48,19 +85,22 @@ export const jsToAqua = (
   );
 
   return `${
-    typeDefs ?? ""
-  }\n\nfunc ${funcName}() -> ${type}:\n    <- ${value}\n`;
+    typeof typeDefs === "string" ? `${typeDefs}\n\n` : ""
```

Comment:
What... why?

When `typeDefs` are `string` and when not? Why silently ignore non-string `typeDefs`? Why event accept non-string `typeDefs`?

Reply:
In aqua standalone type definitions are required for any objects. This means that if the function infered some object - there will be some standalone type definitions. If there were some type defenitions generated - we want to insert them in to the file and separate them from other stuff with new lines. `typeDefs` is either string or undefined so by saing `typeof typeDefs === "string"` I mean `if there are any typeDefs`. e.g. if value is a scalar then there are no separate type definitions and typedefs === undefined